src/assets/netteForms.js

```diff
@@ -27,17 +34,31 @@
 
 var Nette = {};
 
+
+/**
+ * This is necessary to avoid a possible memory leak
+ * in the addEvent function
+ */
+function getEventHandler(callback) {
+	return function (evt) {
+		callback.call(this, evt || window.event);
```

Comment:
Couldn't this break https://github.com/jahudka/forms/blob/dca62a0029af4b558aa1c7572968fc274a5dc11d/src/assets/netteForms.js#L580 ?